boards/px4/fmu-v5x/ssrc.px4board

```diff
@@ -11,6 +11,20 @@ CONFIG_SYSTEMCMDS_REFLECT=n
 CONFIG_SYSTEMCMDS_SD_BENCH=n
 CONFIG_SYSTEMCMDS_SD_STRESS=n
 CONFIG_SYSTEMCMDS_SERIAL_TEST=n
+CONFIG_DRIVERS_CAMERA_CAPTURE=n
+CONFIG_DRIVERS_CAMERA_TRIGGER=n
+CONFIG_DRIVERS_CAMERA_FEEDBACK=n
+CONFIG_MODULES_ROVER_POS_CONTROL=n
+CONFIG_MODULES_SIH=n
+CONFIG_SYSTEMCMDS_NETMAN=n
+CONFIG_SYSTEMCMDS_REFLECT=n
+CONFIG_COMMON_DIFFERENTIAL_PRESSURE=n
```

Comment:
I think we will need some differential pressure sensor with fixed wings, so if possible leave this one